fvm/src/call_manager/default.rs

```diff
@@ -186,6 +186,13 @@ where
                 method,
                 params: params.as_ref().map(Into::into),
                 value: value.clone(),
+                gas_limit: std::cmp::min(
```

Comment:
Ah, actually, this should just be "gas". We only use milligas internally. Just round-up.